Cluster/Why is my node degraded.md

```diff
@@ -0,0 +1,53 @@
+
+### Network probe failed
+Go to [winsock error codes](https://learn.microsoft.com/en-us/windows/win32/winsock/windows-sockets-error-codes-2) and see what can trigger the reported error.
+You might have found `0x80072747: WSAENOBUFS`. If that is the case, [check for port exhaustion;](https://learn.microsoft.com/en-us/troubleshoot/windows-client/networking/tcp-ip-port-exhaustion-troubleshooting)
+If you happen to find a process responsible for this condition, [restarting the process](#stop-the-rogue-process) can alleviate port exhaustion since that will free up ports allocated by the process.
+### Memory probe failed
+The node might not have enough memory to perform the requested operation. There might be a leaking process, or the node could be overloaded. 
+If you have access to the node, Investigate what is causing the memory exhaustion problem inside the node. Open task manager and check the memory usage; you might find a process using excesive memory, or multiple spawn processes; 
+if that is the case, you would need to [stop the process.](#stop-the-rogue-process) <br/>
+If the node is not responding or there is no apparent cause for this failure to happen, [try restarting the node.](#restart-the-node)
+
+### Disk Probe
+> This is reported from the root disk where fabric bits are installed.
+
+This problem gets automatically mitigated after 15 minutes of solid failures. If the node doesn't restart and you keep seeing this error, it could mean that the problem is transient and it is possible that the node has a bad disk or there might be an ongoing issue causing disk unavailability. <br/>
+Open the Windows Event Viewer. Inside Windows Logs/System You might be able to see some warnings/errors where the disk is the source. Use [chkdsk](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk?tabs=event-viewer) to check and try to repair the disk.<br/> 
+If the disk is healthy, there could be a problem with the system and [restarting the node](#restart-the-node) could help to mitigate the issue.
+
+# Mitigations
+
+## Stop the rogue process
+If the reason for the failure is a rogue process whose behavior is impacting fabric, stop the process. You can do it from the task manager by right clicking on the process and then click on **End task**. If you are using the command line for diagnose, you can also run
+`taskkill /f /pid <rogue-process-pid>`
+> If you can't see the process id in task manager, right click in one of the task manager column headers and select the PID option.
+
+## Restart the node 
+**WARNING:** Make sure restarting the node won't cause problems to fabric. If the node to restart is a seed node, make sure the remaining count of alive seed nodes is **greater** than the total count of seed nodes divided by two. 
+- Try to restart the node by using Restart-ServiceFabricNode; however, this command could fail to restart the node since it is already in a degraded state.
```

Comment:
Yes, the intent is to first try to restart the process since it is likely that one of the hosted applications is responsible of the condition, if that doesn't alleviate the problem then restarting the machine would be necessary. Thanks for the feedback, this clarification was missing.